pkg/vpcmodel/textOutput.go

```diff
@@ -40,7 +41,11 @@ func headerOfAnalyzedVPC(uc OutputUseCase, vpcName, vpc2Name string, c1 *VPCConf
 		}
 		return fmt.Sprintf("Connectivity per subnet for VPC %s\n", vpcName), nil
 	case SubnetsDiff, EndpointsDiff:
-		return fmt.Sprintf("Connectivity diff between VPC %s and VPC %s\n", vpcName, vpc2Name), nil
+		header := fmt.Sprintf("Connectivity diff between VPC %s and VPC %s", vpcName, vpc2Name)
+		if !diffSameUID {
+			header += " (note that the compared VPCs are of different UIDs)"
```

Comment:
Is this addition to the header really needed?
The user just provided us with two configs to compare, a single VPC in each.
He/she likely knows whether these VPCs have the same CRN or not.
@adisos , @kyorav , any thoughts here?

Reply:
I don't know the context, but based on your description I don't see how this information can be useful

Reply:
Could be result of a typo? Couldn't the report is shown to someone else? I say better safe than sorry.
Anyways, no big deal this way or another. I'll accept the majority vote ;-)

Reply:
I also think this addition is not required.

Reply:
ok, removed.